### ISSUE TITLE
Data misalignment on two laptops

### ISSUE THREAD

**Issue by swashah** [dhalperi/linux-80211n-csitool-supplementary#257]
Hi, I have two laptops in monitor mode and third laptop is injecting packets. Two laptops are very close to each other and when I analyze the CSI amplitude in matlab data seems to be similar (as it should be logically) misalligned. Is there any way to align the data measured by two laptops based upon time stamp or any thing else. Thanks

**Comment by awelkie** [comment]
What do you mean by "misaligned"? Could you provide some plots or data demonstrating this misalignment?

**Comment by swashah** [comment]
@awelkie : I have attached a figure,as you can see the data is similar but there is no sort of synchronization?
![untitled](https://cloud.githubusercontent.com/assets/23249039/20459755/7a70a4c6-af21-11e6-9ff8-e42d6612b726.png)

**Comment by awelkie** [comment]
Are you plotting amplitude or just the real value? And is the x-axis subcarrier?

**Comment by swashah** [comment]
@awelkie : I am plotting abs(csi) i.e. amplitude and x-axis is number of packets. Graph is for single sub-carrier no. 14. so these are variations of sub-carrier 14 for 45 injected packets.

**Comment by swashah** [comment]
@awelkie : any idea or suggestion> Did you get what I mean?

**Comment by awelkie** [comment]
Yeah, I understand what you mean by misalignment now, but unfortunately I don't have any suggestions. I thought you might be talking about differences between subcarriers, but that's not the case.

**Comment by swashah** [comment]
@awelkie : No worries , thanks for your time

**Comment by hot-dog** [comment]
@swashah Is the delay(packet number between two peaks) fixed? If it is, then the misalignment should not be a matter. I think this misalignment is caused by one monitor ignores some packets in the beginning, it's just a guess, i don't know the true reason either.

**Comment by swashah** [comment]
@hot-dog : I can't get what do you mean by 'delay fixed'. Can you elaborate a bit?

**Comment by hot-dog** [comment]
@swashah I mean the gaps described by the green line.
![7a70a4c6-af21-11e6-9ff8-e42d6612b726](https://cloud.githubusercontent.com/assets/5945046/20513953/2e6092c2-b0c4-11e6-8400-6c73fc5c029c.png)

**Comment by swashah** [comment]
@hot-dog : Apparently they are, but I have not checked in matlab. I will do in the morning and check this out....
